bitcoin/src/blockdata/script/mod.rs

```diff
@@ -600,9 +600,7 @@ pub(super) fn bytes_to_asm_fmt(script: &[u8], f: &mut dyn fmt::Write) -> fmt::Re
     while let Some(byte) = iter.next() {
         let opcode = Opcode::from(*byte);
 
-        let data_len = if let opcodes::Class::PushBytes(n) =
-            opcode.classify(opcodes::ClassifyContext::Legacy)
-        {
+        let data_len = if let Some(n) = opcode.push_bytes() {
```

Comment:
In 19c5927b191ae9d892a2a986829362f878847312:

This new code is wrong because it does not consider PUSHDATA{1,2,4}. We should also decide whether numeric opcodes should count as "pushes".

Reply:
I reread the changes and I believe this mirrors the current behaviour. The `Class::PushBytes(n)` only contains values for `n` between 0 and 75 inclusive.

Reply:
PUSHDATA{1,2,4} is handled lower in the function.

Reply:
Interesting. Indeed it does match the current behavior. If anybody ever used these pushbytes opcodes that would be a pretty big footgun..